app/assets/javascripts/components/multi-section-viewer.js

```diff
@@ -0,0 +1,41 @@
+function MultiSectionViewer (container) {
+  this.$container = container
+  this.$dynamicSection = container.querySelector('.js-dynamic-section')
+}
+
+MultiSectionViewer.prototype.init = function () {
+  var actions = document.querySelectorAll('[data-toggle="multi-section-viewer"][data-target="' + this.$container.id + '"]')
+  var $module = this
+
+  actions.forEach(function (action) {
+    action.addEventListener('click', function (event) {
+      $module.showStaticSection(event.target.dataset.targetSection)
+    })
+  })
+}
+
+MultiSectionViewer.prototype.hideAllSections = function () {
+  var sections = this.$container.querySelectorAll('.app-c-multi-section-viewer__section')
+
+  sections.forEach(function (section) {
+    section.style.display = 'none'
```

Comment:
From discussion with @alex-ju: the idea here is to separate behaviour from look-and-feel. I think the problem we'd have here is making the `--visible` have precedence over the default class, as they'd both be on the same element. We do use this approach elsewhere; the only reason for having a class is to prevent flickering when the component is initialised.

Reply:
govuk-frontend achieve this by the --hidden modifer being defined later in the css: https://github.com/alphagov/govuk-frontend/blob/master/src/components/tabs/_tabs.scss#L119-L121 - which I guess is a cornerstone of BEM

Reply:
I had a quick go at this, but it really does introduce a lot of bloating without any obvious benefit, so I'm going to stick with what it is now. Quite happy to change it if it becomes necessary :-).